config/default.js

```diff
@@ -21,10 +21,22 @@ module.exports = {
     },
     directory: path.join(__dirname, '../content-types'),
     messages: {
+      format: {
+        revision: 'Revision must be a number',
```

Comment:
This crap is ugly and I don't like it and we really _really_ need to get [i18n](https://www.npmjs.com/package/i18n) in to do away with all of this